docs/rules/order.md

```diff
@@ -3,7 +3,7 @@
 Enforce a convention in the order of `require()` / `import` statements. The order is as shown in the following example:
 
 ```js
-// 1. node "builtins"
+// 1. node "builtin"
```

Comment:
this is a comment, so changing this doesn't alter it's copy-pasteability, but also it's correct as-is, as a plural. it's describing "node builtins", plural.

Reply:
It's not valid for version `2.8.0`:
```
  2:1  error  Incorrect configuration of the rule: Unknown type `"builtins"`  import/order
```

Is looks like copy-pastable cause it's wrapped with quotes, and all other values are valid. Also it looks very similar to real valid value from documentation, do if I have already read the doc I may open it and copy-paste values from first section. It's just about usability. If `"builtins"` should be a correct value, we have a bug inside plugin and it's undocumented feature.

Reply:
ohhh i'm sorry, i think i get what you mean. yes, "builtin" is the config value, so this is a good change. Maybe `node "builtin" modules`?

Reply:
ok, i'll change it now